external/configuration-guides/vpn.md

```diff
@@ -97,31 +97,31 @@ Set up a ProxyVM as a VPN gateway using iptables and CLI scripts
 This method is more involved than the one above, but has anti-leak features that also make the connection _fail closed_ should it be interrupted.
 It has been tested with Fedora 23 and Debian 8 templates.
 
-1. Create a new VM, name it, click the ProxyVM radio button, and choose a color and template.
+1. Create a new VM, name it, choose 'Provide network' and choose a color and template and sys-Firewall in networking.
 
    ![Create\_New\_VM.png](/attachment/wiki/VPN/Create_New_VM.png)
 
    Note: Do not enable NetworkManager in the ProxyVM, as it can interfere with the scripts' DNS features.
    If you enabled NetworkManager or used other methods in a previous attempt, do not re-use the old ProxyVM...
    Create a new one according to this step.
 
-   If your choice of TemplateVM doesn't already have the VPN client software, you'll need to install the software in the template before proceeding.
+   If your choice of TemplateVM doesn't already have the VPN client software, you'll need to install the software in the template before proceeding.just install openvpn in template. Default Fedora 30 or higher don't need this step.
    Disable any auto-starting service that comes with the software package.
    For example for OpenVPN.
 
        sudo systemctl disable openvpn.service
 
-   You may also wish to install `nano` or another simple text editor for entering the scripts below.
-
 2. Set up and test the VPN client.
    Make sure the VPN VM and its TemplateVM is not running.
    Run a terminal (CLI) in the VPN VM -- this will start the VM.
    Then create a new `/rw/config/vpn` folder with.
 
        sudo mkdir /rw/config/vpn
-
-   Copy your VPN config files to `/rw/config/vpn`.
-   Your VPN config file should be named `openvpn-client.ovpn`) so you can use the scripts below as is without modification.
+   Now run Browser in Disposable VM and download vpn configuration files from your provider. Copy these into your VPN VM. Your VPN configuration files in can be seen in QubesIncoming in VPN VM (You can extract here if its a zip file). Now open one of the configuration file and keep it open.
+   Now run a terminal in VPN VM. run here
+   sudo gedit /rw/config/vpn/openvpn-client.ovpn
```

Comment:
You need to indent it with 4 spaces and also surround with blank lines, to have it properly formatted as code.